testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/AuthServerTestEnricher.java

```diff
@@ -70,28 +70,28 @@
     private Event<StopContainer> stopContainerEvent;
 
     public static final String AUTH_SERVER_CONTAINER_DEFAULT = "auth-server-undertow";
-    private static final String AUTH_SERVER_CONTAINER_PROPERTY = "auth.server.container";
+    public static final String AUTH_SERVER_CONTAINER_PROPERTY = "auth.server.container";
```

Comment:
Why do you need to change this?

Reply:
There was a mix of private and public fields there (I believe anyone that needed them was simply changing the access modifier from `private` to `public`). I just unified them to `public`. Since all of them are effectively constants (`static final`), it's safe to use them by calling fields.

Reply:
ok, I have no objections to make it public. Thanks for the explanation ;)